rapids-cmake/cmake/support_conda_env.cmake

```
@@ -41,6 +41,13 @@ the behavior of conda when it builds projects.
 The `target_name` target will add the required compile flags to ensure debug builds
 are generated with `-O0` instead of the conda env default of `-O2`.
 
+.. versionadded:: v25.06.00
+
+The `-ffile-prefix-map` is now passes to remap absolute paths starting
+with `$ENV{PREFIX}` to paths relative to it. This ensures paths baked
+into binaries are relative to the environment prefix a usually
+reasonable default. It also bypasses the need to rewrite these paths.
```

Rob, how does this look?

---

Suggesting some minor tweaks.
```suggestion
.. versionadded:: v25.06.00

The flag `-ffile-prefix-map` is now passed to remap absolute paths
starting with `$ENV{PREFIX}` to paths relative to it. This ensures
paths baked into binaries are relative to the environment prefix.
This prevents conda from rewriting these paths when the package is
installed.
```

---

Can we re-phrase the first sentence to:
```
The flag `-ffile-prefix-map` is now passed to remap absolute paths starting with `$ENV{PREFIX}` to relative paths starting from `$ENV{PREFIX}` when contained in pre-processor macros like `__FILE__`. 
```

I want to make it clear that this flag doesn't impact how other compiler flags  ( e.g `-I` or `-isystem` ) are parsed, but just impacts code generation.

---

Rob, took a slightly different approach, but tried to capture the spirit of this suggestion